TLSharp.Tests/TLSharpTests.cs

```diff
@@ -192,7 +191,7 @@ public virtual async Task SendMessageToChannelTest()
 
             await client.ConnectAsync();
 
-            var dialogs = (TLDialogs) await client.GetUserDialogsAsync();
+            var dialogs = (TLDialogs)await client.GetUserDialogsAsync();
```

Comment:
no need for this cosmetic change, please undo (or separate this change into a different commit or PR)